src/tiled/changetilewangid.cpp

```diff
@@ -0,0 +1,91 @@
+#include "changetilewangid.h"
+
+#include "tilesetdocument.h"
+#include "tile.h"
+
+#include <QCoreApplication>
+
+using namespace Tiled;
+using namespace Internal;
+
+ChangeTileWangId::ChangeTileWangId()
+    : mTilesetDocument(nullptr)
+    , mWangSet(nullptr)
+    , mMergeable(false)
+{
+    setText(QCoreApplication::translate("Undo Commands", "Change Tile WangId"));
+}
+
+ChangeTileWangId::ChangeTileWangId(TilesetDocument *tilesetDocument, WangSet *wangSet, Tile *tile, WangId wangId)
+    : mTilesetDocument(tilesetDocument)
+    , mWangSet(wangSet)
+    , mMergeable(true)
+{
+    setText(QCoreApplication::translate("Undo Commands", "Change Tile WangId"));
+    if (mWangSet)
```

Comment:
When this happens, I think it would indicate an error in the code calling the constructor. You can consider writing `Q_ASSERT(wangSet);` at the start of the constructor instead so we can catch that.